Orange/widgets/data/oweditdomain.py

```diff
@@ -662,6 +673,168 @@ def on_label_selection_changed(self):
         self.remove_label_action.setEnabled(bool(len(selected)))
 
 
+class GroupLessFrequentItemsDialog(QDialog):
+    """
+    A dialog for group less frequent values.
+    """
+    def __init__(
+            self, variable: Categorical, data: Union[np.ndarray, List],
+            selected_attributes: List[str], parent: QWidget = None,
+            flags: Qt.WindowFlags = Qt.Dialog, **kwargs
+    ) -> None:
+        super().__init__(parent, flags, **kwargs)
+        self.variable = variable
+        self.data = data
+        self.selected_attributes = selected_attributes
+
+        # grouping strategy
+        self.selected_radio = radio1 = QRadioButton("Group selected values")
+        self.frequent_abs_radio = radio2 = QRadioButton(
+            "Group values with less than"
+        )
+        self.frequent_rel_radio = radio3 = QRadioButton(
+            "Group values with less than"
+        )
+        self.n_values_radio = radio4 = QRadioButton(
+            "Group all except"
+        )
+
+        # if selected attributes available check the first radio button,
+        # it is most probable option
+        if selected_attributes:
```

Comment:
What about even disabling (but not hiding - let the user know it's there!) the first option when there is no selection?